src/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -0,0 +1,66 @@
+<?php namespace October\Rain\Database\Eloquent\Concerns;
+
+use Illuminate\Support\Str;
```

Comment:
We should use our Str helper, not Laravel's.

Reply:
Modified the Str helper used to the October one :)